SDK/AppCenter/Microsoft.AppCenter.Windows.Shared/Storage/Storage.cs

```diff
@@ -51,7 +51,7 @@ internal class LogEntry
         /// <summary>
         /// Creates an instance of Storage.
         /// </summary>
-        public Storage() : this(DefaultAdapter(), Constants.AppCenterDatabasePath)
+        public Storage() : this(DefaultAdapter(), Path.Combine(Constants.AppCenterFilesDirectoryPath, Constants.AppCenterDatabasePath))
```

Comment:
Is it the same as on develop now? Why it doesn't covered by tests? What happens with existing DBs? Did you checked it?

Reply:
1) Develop is a bit ambiguous - db is created without explicit prefix\localAppData being appended, even though it is created in LocalAppData (`%LocalAppData%\Packages\{package}\LocalState\Microsoft.AppCenter.Storage`, and the deletion uses prefix explicitly (`var prefix = _databaseDirectory == null ? Constants.LocalAppData : "";`)
2) Not sure about coverage tbh
3) Effectively the code in this PR is the same as we currently have in develop.
I think previous implementation (SQLite-net-pcl) handled db file path internally, without us having to specify folder explicitly, while the new implementation (SQLitePCLRaw) expects to get fully qualified path, otherwise it tries to create file locally (which caused exception)
4) Checked on dev:
Explicit prefix - LocalAppDb used (Correct)
Without prefix (implicit) - LocalAppDb used (Correct)
And on this PR:
Explicit prefix - LocalAppDb used (Correct)
Without prefix - **Incorrect** - attempt to create file locally.